config/services/common.yml

```diff
@@ -25,3 +25,7 @@ services:
     PrestaShopBundle\DependencyInjection\RuntimeConstEnvVarProcessor:
         public: false
         tags: ['container.env_var_processor']
+
+    PrestaShop\PrestaShop\Core\Hook\HookModuleFilter:
+      arguments:
+        - !tagged core.hook_module_exec_filter
```

Comment:
```suggestion
```
Since the service is already defined in `../../src/PrestaShopBundle/Resources/config/services/core/common.yml` which is loaded in this file you don't need to duplicate the definition here

Reply:
it doesn't work anymore in FO if I remove this service definition... the service `PrestaShop\PrestaShop\Core\Hook\HookModuleFilter` is unfound